schemas/db-schema.sql

```diff
@@ -14,9 +14,11 @@ username VARCHAR(30)
 
 CREATE TABLE tasks
 (
-task_id INT(6) NOT NULL PRIMARY KEY AUTO_INCREMENT,
+task_id INT(6) NOT NULL AUTO_INCREMENT,
 user_id INT(6) NOT NULL,
 task_text VARCHAR(80) NOT NULL,
 task_date DATE,
-task_completion_date DATE NULL DEFAULT NULL
+task_completion_date DATE NULL DEFAULT NULL,
```

Comment:
This is not of much significance, but consider using `DATETIME`. (This is from a business prespective)